public/app/features/trails/shared.ts

```diff
@@ -32,7 +32,8 @@ export const trailDS = { uid: VAR_DATASOURCE_EXPR };
 
 // Local storage keys
 export const RECENT_TRAILS_KEY = 'grafana.trails.recent';
-export const BOOKMARKED_TRAILS_KEY = 'grafana.trails.bookmarks';
+
+export const BOOKMARKED_STATES_KEY = 'grafana.trails.bookmarkedStates';
```

Comment:
any way we can keep the old key and just detect / migrate the old ones?

Reply:
Done!
Simplifies much.